scripts/docker/build_docker.py

```diff
@@ -307,39 +529,66 @@ def __parse_arguments(args_list):
     docker_remote_args_group = parser.add_argument_group(
         'Docker push', 'controlling behavior related pushing dockers to remote repos'
     )
-    docker_remote_args_group.add_argument('--gcr-project', type=str,
-                                          help='GCR billing project to push the images to. If not given, the '
-                                               'built image(s) will not be push to GCR.')
+    docker_remote_args_group.add_argument('--docker-repo', type=str, help='Docker repo to push images to.')
     docker_remote_args_group.add_argument('--update-latest', action='store_true',
-                                          help='also update \"latest\" tag in remote docker repo(s)')
-    # flag to turn off git protection (default mode is refusing to build when there are untracked files and/or
-    # uncommitted changes)
+                                          help=f'also update \"{ProjectBuilder.latest_tag}\" tag in remote docker'
+                                               f'repo(s)')
+    dockers_json_path = os.path.join(Paths.gatk_sv_path, "input_values", "dockers.json")
+    docker_remote_args_group.add_argument('--input-json', type=str, default=dockers_json_path,
+                                          help="Path to dockers.json to use as input")
+    docker_remote_args_group.add_argument('--output-json', type=str, default=dockers_json_path,
+                                          help=f"Path to output updated dockers.json. Set to {Paths.dev_null} to turn "
+                                               "off updates")
+    parser.add_argument("--dry-run", action="store_true",
+                        help="Compute docker images that will be build, but don't actually build or push")
+    parser.add_argument("--skip-dependent-images", action="store_true",
+                        help="Don't build images that depend on targets. Can leave images in an unreproducible state: "
+                             "only use this if you are sure you know what you are doing!")
+    parser.add_argument(
+        '--targets', nargs='*', type=str,
+        help='the sub project docker(s) you want to build (note "all" does not include melt)'
+    )
+    short_git_hash_head = get_command_output("git rev-parse --short HEAD").strip()
+    parser.add_argument('--image-tag', type=str, default=short_git_hash_head,
```

Comment:
I'm open to changing this, but my logic was that a person who is developing a WDL pipeline may want to make and test repeated changes. As they are working, they may eventually decide they like a set of changes and commit them, update their WDL inputs, etc. This default makes that workflow easy. I know I've accidentally messed up while manually specifying my git hashes, and clobbered the wrong image, etc. It also makes automation pretty easy for github actions. I don't see the advantage to adding a date-stamp to the image tag since the hash is already unique, and it prevents the workflow I just described. You're working Tuesday and come back to your work Wednesday and suddenly any changes you make go into a different image tag.

Reply:
I have a vague recollection that we decided to add timestamp so we could reason about the precedence of images without having to consult with the git history. I guess it would be useful to be consistent in the defaults, but I can see CI/CD being a different context, so it can have its own defaults.